CODEOWNERS

```diff
@@ -0,0 +1,3 @@
+# Repo-Level Owners (in alphabetical order). Org-level maintainers are included in the linked team at the end
+# For Org level maintainers, refer to https://github.com/notaryproject/.github/blob/main/CODEOWNERS
+* @duffney @shizhMSFT @Two-Hearts @notaryproject-org-maintainers
```

Comment:
It seems that @notaryproject-org-maintainers isn't correct. Should it be @notaryproject/notaryproject-org-maintainers?